builders/containerBuilder.go

```diff
@@ -0,0 +1,148 @@
+package builders
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/tfexplorer/models/generated/workloads"
+)
+
+// ContainerBuilder is a struct that can build containers
+type ContainerBuilder struct {
+	workloads.Container
+}
+
+// NewContainerBuilder creates a new container builder
+func NewContainerBuilder(nodeID, flist, flistStorage string, networkConnection []workloads.NetworkConnection) *ContainerBuilder {
```

Comment:
you can remove `flistStorage` from here and just use a default value for it that points to `zdb://hub.grid.tf:9900`